DownloadSongs.sh

```diff
@@ -3,11 +3,22 @@
 mkdir -p $HOME/Music/SongLibrary
 cd $HOME/Music/SongLibrary
 
+while getopts ":a" opt; do
+  case $opt in
+    a)
+	aws configure
```

Comment:
We don't need to configure every time we run the script, it makes more sense to me that this step should be done manually, described in the readme in step 2.